src/extension.ts

```diff
@@ -46,6 +47,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 
 	applyUserSettings();
 
+	// call didact to register tutorial
+	await registerTutorialWithDidact(context);
```

Comment:
I'm wondering if we really need to await that the tutorial is registered. I mean the Camel K extension can continue to start even if the tutorial is not registered. What do you think?

Reply:
I think because we are updating the workbench configuration, we need to wait for it to complete before moving on or risk collisions when other extensions also add configurations upon activation

Reply:
ok fine for a first iteration.

this is something that will need to be improved in the future because it is breaking the whole asynchronuous effort that has been done in VS Code to provide a more responsive UI.

In case one extension is providing a huge amount of tutorial, I fear we will notice the slowness.